playbooks/host_vars/ansibullbot-cloud.eng.ansible.com.yml

```diff
@@ -0,0 +1,58 @@
+---
+# Settings for EC2 instance
+botinstance_name: ansibullbot-cloud
```

Comment:
Putting the `botinstance_*` variables in this `host_var` won't have any affect since the play that uses these variables runs against `localhost` since it's talking to AWS, not a particular host.

Reply:
I'm actually using most of these in an inventory file for deploying which is what I've put into the docs, not sure where (or if?) to house that in the repo though.  Should we split out all the actual config info that the Ansible-managed bots have to a separate repo to make this one more instance-neutral, or just make an inventories dir, or something else?